patches/minecraft/net/minecraft/client/Minecraft.java.patch

```diff
@@ -385,12 +393,15 @@
 +         connection.m_129512_(new ServerboundHelloPacket(gameProfile));
           this.f_91009_ = connection;
        } else {
++         if(flag) //For legacy world options, let vanilla handle it.
```

Comment:
Formatting: space before opening parenthesis in `if`-expression. Also, a `FORGE` marker should be added for clarity.
```suggestion
+         if (flag) // FORGE: For legacy world options, let vanilla handle it.
```